pyccel/ast/core.py

```diff
@@ -2519,11 +2519,38 @@ def __new__(cls, arg_vars, *args, **kwargs):
 
```

Comment:
A comment about initialization.

Why do we need to pass `NativeTuple()` as the second argument to the constructor of `TupleVariable`? Are we allowed to pass datatypes other than `NativeTuple()`? Or maybe in the future we might want to?

If not, it would seem more natural to me not to pass it at all.

Reply:
Thanks. You may need to update the example in the docstring now.